debian/control

```diff
@@ -70,8 +70,8 @@ Breaks: libmariadb-client-lgpl-dev,
 Replaces: libmariadb-client-lgpl-dev,
           libmariadb-dev-compat (<< 3.0.0),
           libmariadbclient-dev,
-          libmysqlclient-dev (<< ${source:Version}),
```

Comment:
It feels like the old version dependency was wrong, but I don't understand exactly what this meant.

Can you elaborate in the commit comment what exactly this change would mean for a new version of a mariadb-server-10.6 package that was not true for a mariadb-server-10.6 package *before* your change?

Reply:
Reasoning behind this is in: https://lintian.debian.org/tags/version-substvar-for-external-package. In other words `libmysqlclient-dev` so it should be removed (every version which it does actually) as it's not part of this package.